### ISSUE TITLE
Amount of regularization used when training 'pretty' mesh-rcnn for shape-net dataset

### ISSUE THREAD

**Issue by udaranga3001** [facebookresearch/meshrcnn#64]
Hi @gkioxari ,

According to the paper (and also the code), you have used 0.2 as the coefficient for edge loss when training the 'pretty' model on shape-net dataset.

1.  I would like to know whether the examples you have given in Figure 5 and 6 in the paper are from the same model or a model trained with larger amount of regularization? The reason I asked this question is that when I run the demo with pre-trained weights on shape-net dataset, I notice my meshes are much more noisy (not all the time, but in considerable amount of instances).

2. I would also like to know why you didn't try laplacian loss? I feel it can help to reduce this noise. I agree that it would increase the chamfer loss, but it would improve the quality of the meshes (the prettiness).

**Comment by gkioxari** [comment]
Re 1.: The figures on the paper are consistent with the model parameters we report on the paper. It is a trend in 3D papers show model visualizations that do not correspond to the actual model reported in the paper, in an effort to show pretty results and best numbers. That is unscientific. 

Re 2.: We tried to use the laplacian loss with cotangent cuvature which includes a term that divides by the area of a face. Because our architecture makes drastic changes to the topology and geometry, that led often to faces with very small area (close to 0) and the regularizer would become `nan`. We didn't try the `uniform` version of laplacian smoothing.

**Comment by udaranga3001** [comment]
Thank you for the answers,

Re 1.: I totally agree that the current trend is unscientific. I just wanted to get a clarification regarding the pretty model used in the figures.

Re 2.: I have also had similar negative experience with laplacian loss with cotangent cuvature when training mesh deforming models. But the uniform version worked and gave me better results (in terms of the prettiness). I will try the uniform version with mesh-rcnn and if I see an improvement I will add a comment here.
